app/views/home/_retailers_list.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="row retailers text-center">
+  <div class="col-md-4">
+    <%= image_tag('wayfair-logo.png', class: 'retailer-logo') %>
+  </div>
+  <div class="col-md-4">
+    <%= image_tag('houzz-logo.png', class: 'retailer-logo') %>
+  </div>
+  <div class="col-md-4">
+    <%= image_tag('hayneedle-logo.svg', class: 'retailer-logo') %>
+  </div>
```

Comment:
@hiattp @birdytripler these retailers are hardcoded for now since I'm not sure about something. So my questions are
1. Should we have seeds for creating all retailers' data such as their logo, name, etc? since these data would not be changed at all.
2. Should I upload retailers' logo into S3?